change/@fluentui-react-positioning-a42182b3-5c4f-4c1a-bb0c-fa1815d2c14c.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "feat(createArrowStyles): Add utility function to create styles for an arrow element",
```

```suggestion
  "comment": "Add utility function to create styles for an arrow element",
```

We probably should not use Angular commit notation in changelog 🤔